src/shell/xdg/inner.rs

```diff
@@ -41,26 +36,18 @@ where
         }
     }
 
-    fn remove_global(state: &mut D, _: &mut ConnectionHandle, _: &QueueHandle<D>, name: u32) {
-        if state
-            .xdg_shell_state()
-            .xdg_wm_base
-            .as_ref()
-            .filter(|(global_name, _)| global_name == &name)
-            .is_some()
-        {
-            todo!("XDG shell global destruction")
-        }
+    fn remove_global(_: &mut D, _: &mut ConnectionHandle, _: &QueueHandle<D>, _: u32) {
+        // TODO: How to handle "defunct surfaces" error
```

Comment:
That's not really a problem tbh, removal of the `xdg_wm_base` global is definitely in the "will never happen" category.